cvat-ui/src/components/organization-page/top-bar.tsx

```diff
@@ -319,93 +345,53 @@ function OrganizationTopBar(props: Props): JSX.Element {
-                                            ) : null}
-                                        </Col>
-                                    </Row>
-                                ))}
-                                <Form.Item>
-                                    <Button className='cvat-invite-more-org-members-button' icon={<PlusCircleOutlined />} onClick={() => add()}>
-                                        Invite more
-                                    </Button>
-                                </Form.Item>
-                            </>
+            <Row className='cvat-oranization-page-filters-wrapper' justify='space-between'>
+                <Col>
+                    <Input.Search
+                        enterButton
+                        onSearch={(phrase: string) => {
+                            onApplySearch(phrase);
+                        }}
+                        defaultValue={query.search || ''}
+                        className='cvat-oranization-page-search-bar'
+                        placeholder='Search ...'
+                    />
+                </Col>
+                <Col>
+                    <SortingComponent
+                        visible={visibility.sorting}
+                        onVisibleChange={(visible: boolean) => (
+                            setVisibility({ ...defaultVisibility, sorting: visible })
+                        )}
+                        defaultFields={query.sort?.split(',') || ['-ID']}
+                        sortingFields={['User', 'Role']}
+                        onApplySorting={onApplySorting}
+                    />
+                    <FilteringComponent
+                        value={query.filter}
+                        predefinedVisible={visibility.predefined}
+                        builderVisible={visibility.builder}
+                        recentVisible={visibility.recent}
+                        onPredefinedVisibleChange={(visible: boolean) => (
+                            setVisibility({ ...defaultVisibility, predefined: visible })
+                        )}
+                        onBuilderVisibleChange={(visible: boolean) => (
+                            setVisibility({ ...defaultVisibility, builder: visible })
                         )}
-                    </Form.List>
-                </Form>
-            </Modal>
+                        onRecentVisibleChange={(visible: boolean) => (
+                            setVisibility({
+                                ...defaultVisibility, builder: visibility.builder, recent: visible,
+                            })
+                        )}
+                        onApplyFilter={onApplyFilter}
+                    />
+                </Col>
+            </Row>
+            <InvitationModal
+                onInvite={onInvite}
+                onCancelInvite={onCancelInvite}
+                visibleInviteModal={visibleInviteModal}
+            />
```

Comment:
To avoid extra rendering: 

```suggestion
            visibleInviteModal && (
                <InvitationModal
                    onInvite={onInvite}
                    onCancelInvite={onCancelInvite}
                    visibleInviteModal={visibleInviteModal}
                />
        )
```

Reply:
Applied, also removed visibleInviteModal param as its not useful anymore